README.md

```diff
@@ -17,17 +17,41 @@ jobs:
       - name: Create tag and release
         uses: silverstripe/gha-tag-release@v1
         with:
+          skip_gauge_release: true
           tag: 1.2.3
-          release: true
+          release: false
 ```
 
 ### Inputs
 
-#### tag (required)
-The tag to create e.g. 1.2.3
+### Latest local sha
+Required if not skipping gauge release.
+
+The result of `$(git rev-parse HEAD)`.
+
+`latest_local_sha: f22dbc6ec6118096c8ccccee1ca0074bfb2f2291`
+
+### Skip gauge release
+Whether to skup gauging the release. Gauging the release will only allow tagging if all of the following are true:
+
+- The branch this action is run against is a patch branch (e.g. `1.2`)
+- The `latest_local_sha` input matches the `github.sha` GitHub actions variable
+- The `latest_local_sha` input matches the current latest commit sha on this branch
+- There's an existing stable semver patch tag for this branch already (i.e. for branch `1.2` there must be a `1.2.<digit>` tag)
+- There are commits on the branch that warrant a new patch release and weren't included in the latest patch release for this branch
+
+Gauging the release also identifies the correct next patch tag, and uses that to tag the release.
```

Comment:
Mention what the default value for skip_gauge_release is

Reply:
Done